### DISCUSSION TITLE
mihomo如何设置面板访问密码

### DISCUSSION THREAD

**Discussion by jiashu1024** [MetaCubeX/mihomo#1017] [Q&A] [answered]
由于我在服务器上使用 mihomo 用于加速，mihomo 暴露了 9090 端口，我用过 yacd 面板可以用来连接 9090 端口，从而控制节点切换，tun 转发等功能。但是默认情况下是不需要密码的，由于在公网部署，有没有哪里可以配置这个访问密码

**Comment by RealTong** [comment, marked as answer]
```
external-controller: :9090
secret: "YOUR PASSWORD"
```

**Comment by jiashu1024** [comment]
> ```
> external-controller: :9090
> secret: "YOUR PASSWORD"
> ```
感谢
